### DISCUSSION TITLE
Help with existing transcripts

### DISCUSSION THREAD

**Discussion by fbv000** [ggml-org/whisper.cpp#2145] [Q&A]
AI does not seem to work with existing transcripts (whisper files). 
Here is my workflow:

- open an audio file.
- generate transcript
- save .whisper file

Later on open the .whisper file and run a prompt. The prompt does not work, it does not seem to recognize the existing transcript.

Any suggestions?

**Comment by ulatekh** [comment]
What is a .whisper file? What generates it? I've never seen such a thing using any Whisper-related project.
